modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/common/components/HighPriorityContacts/SetupHighPriorityContact.jsx

```diff
@@ -34,6 +47,7 @@ const SetupHighPriorityContact = ({
 			contactsCategory: {
 				key: _filter,
 				name: `${filter}`,
+				role: getContactRoleByFilter(`${filter}`),
```

Comment:
Do we really need template strings here?

Reply:
thanks!